### ISSUE TITLE
Evaluate appropriate baseline layout analysis models

### ISSUE THREAD

**Issue by kdutia** [climatepolicyradar/navigator#36]
- [x] Choose ~two~ one simple baseline model~s~ to assess based on model types and datasets they're trained on
- [x] Pick performance metric(s) (IOU? Better quality text?)
- [x] Assess performance of baseline models on the test set

**Comment by kdutia** [comment]
I suggest we start with the evaluation metric of pixel-wise MAP @ 0.5 IOU, as this will allow us to compare our results to some of the literature. Write up of different evaluation metrics used here https://www.notion.so/climatepolicyradar/Document-layout-analysis-for-PDFs-literature-review-c6be062dcc14468287d1c04c05dbdd7a#671efa59c5484107962b2ac7fcb5b86b

**Comment by kdutia** [comment]
The first baseline model is the largest faster-RCNN model from the [layout parser model hub](https://layout-parser.readthedocs.io/en/latest/notes/modelzoo.html). Struggling to find a second which is trained, meaning that we'd need training data.

**Comment by kdutia** [comment]
Ended up testing one baseline model, as it was the only pretrained model I could find on a suitable dataset. Measured performance using mean average precision @ 50% IOU, although after discussions with @chrisaballard we agreed we'd like to move to a more text-based metric in the shorter term, as a) our approach may not sit under the umbrella of object detection and b) what we really care about is the classification of text passages.

The model evaluation is in [this notebook](https://github.com/climatepolicyradar/pdf-parsing-evaluation/blob/main/research/model-evaluation.ipynb). In summary:

* The model performs too poorly to be immediately implemented into the product.
* Sometimes text is missed by the bounding boxes predicted by the model. We'd need to decide on a heuristic to use to label this text with a document layout element.
* The model performs worse on legislative than executive texts. This is partly likely due to labelling - sometimes it's difficult to tell whether a set of legal articles should be identified as text passages or a list. This indicates that we should start simpler, with text block detection, as this is what we need for text display and retrieval.
* The model performs poorly on noisy scans or documents with non-white backgrounds. To identify and improve this for future models we should consider labelling types of pages (e.g. noisy/non-white background/contents page/page left blank) whilst doing the layout element analysis, and then potentially some image-preprocessing in the pipeline.
